invenio_access/models.py

```diff
@@ -53,7 +53,15 @@ class ActionNeedMixin(object):
 
     @classmethod
     def create(cls, action, **kwargs):
-        """Create new database instance from action need."""
+        """Create new database instance from action need.
+
+        :param action: A object containing a method equal to ``'action'`` and
+            a value.
+        :param argument: The action argument. If it's ``None`` then, if exists,
+            the ``action.argument`` will be taken. In the worst case will be
+            set as ``None``. (Default: ``None``)
```

Comment:
Much better! But in that context, it isn't clear what situation 'otherwise' refers to... I would put it like that:

`If the ``action.argument`` does not exist, ``None`` will be set as argument for the new action need.`